README.md

```diff
@@ -5,38 +5,43 @@
 
 > Rust & RIOT combined for ergonomic embedded development
 
-This is an experimental project to provide a nice base OS for embedded
-development on low-end IoT devices (with some kilobytes of RAM/flash, think
-Cortex-M). It combines the awesome Rust embedded ecosystem with
-[RIOT](https://github.com/RIOT-OS/RIOT).
+RIOT-rs is a project aiming to provide a general-purpose
+operating system adequate for cybersecure, memory-safe, low-power IoT.
 
-**This is highly experimental. Expect heavy changes and breakage!**
+RIOT-rs is based on Rust from the ground up, and uses formal verification for critical modules. 
+For a more verbose rant, see this
+[manifesto](https://future-proof-iot.github.io/RIOT-rs/dev/manifesto.html).
 
-If you're looking for a more production ready way of writing RIOT applications
-in Rust, check out [riot-wrappers](https://gitlab.com/etonomy/riot-wrappers).
+Overall, RIOT-rs aims for a 'batteries-included' experience, on par
```

Comment:
"RIOT-rs aims for" may be rewritten in firmer terms. Maybe "RIOT-rs will give you"? or "gives you".

Reply:
thanks for the suggestion, now incorporated in the text.